tests/core/arch-arm/armrelocator.c

```diff
@@ -236,6 +236,8 @@ branch_scenario_execute (BranchScenario * bs,
 {
   gsize i;
   const cs_insn * insn = NULL;
+  gboolean same_content;
+  gchar * diff = NULL;
```

Comment:
```suggestion
  gchar * diff;
```

(Seems like a redundant initialization.)